tests/TESTS/common/fixtures.py

```diff
@@ -181,10 +181,12 @@ async def release(self, client: BleakClient) -> None:
                     except BleakError:
                         pass
                     attempts -= 1
+            """
```

Comment:
if the code is redundant, remove it or leave a comment explaining why you left it (presumably because you need someone with a mac to test it?

Reply:
Perhaps. I'll add a comment if you like, but I don't expect this to remain after review.

Reply:
I removed it: 5ec12537b67516d54246a858a4a75fd340aa5f5a